_posts/2017-09-03-variables-in-inspector.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Making Variables Visible in Inspector
+date:   2017-09-03
+categories: Tips&Tricks
+tags: inspector code beginner
+excerpt:
+mathjax: false
+author: BaconGamer88
+---
+
+Beginner programmers in Duality may have begun to create their scripts, only to wonder why they can't read/write variables in the 
+inspector. In Unity, you would declare the variable as public or put `[SerializeField]` in front of the variable. However, both approaches 
+will not work in Duality. So how would you go about doing this? Well, this snippet shows how to do this with a float variable, but it will 
+work with any variable type, including integers, strings and booleans.
```

Comment:
I might add here, that any non-builtin types can be exposed too. (I know that logically "any variable type" covers these, but it's a bit more straightforward. Or we can omit the "including" part.)